src/components/MobileWalletDialog/routes/DeleteWallet/Backup.tsx

```diff
@@ -0,0 +1,60 @@
+import { Button, Heading, Stack } from '@chakra-ui/react'
+import React, { useCallback } from 'react'
+import { useTranslate } from 'react-polyglot'
+import { useHistory } from 'react-router'
+import { MobileWalletDialogRoutes } from 'components/MobileWalletDialog/types'
+import { DialogBackButton } from 'components/Modal/components/DialogBackButton'
+import { DialogBody } from 'components/Modal/components/DialogBody'
+import { DialogFooter } from 'components/Modal/components/DialogFooter'
+import { DialogHeader, DialogHeaderLeft } from 'components/Modal/components/DialogHeader'
+import { SlideTransition } from 'components/SlideTransition'
+import { RawText } from 'components/Text'
+import { useModal } from 'hooks/useModal/useModal'
+
+// TODO: This is placeholder content follow up PR will implement this correctly
```

Comment:
Is this comment still valid?

Reply:
Yea i'm leaving the boiler plate in for when we spike to see if we can actually do a backup on a wallet.